tests/suite-integration/PluginFlyvemdmAgent.php

```diff
@@ -196,16 +211,6 @@ public function testInvalidEnrollAgent(array $data, $expected) {
       $this->boolean($agent->isNewItem())
          ->isTrue(json_encode($_SESSION['MESSAGE_AFTER_REDIRECT'], JSON_PRETTY_PRINT));
       $this->array($_SESSION['MESSAGE_AFTER_REDIRECT'][ERROR])->contains($expected);
-
-      // Check registered log
-      // previous enrolment could generate a new log
-      $invitationLog = new \PluginFlyvemdmInvitationlog();
-      $fk = \PluginFlyvemdmInvitation::getForeignKeyField();
-      $inviationId = $invitation->getID();
-      $expectedLogCount += count($invitationLog->find("`$fk` = '$inviationId'"));
-
-      $total = $dbUtils->countElementsInTable($invitationlogTable);
-      $this->integer($total)->isEqualTo($expectedLogCount);
```

Comment:
this code checks that no log entry is added, but it may malfunction because of parallelization of tests (an other test may add an entry in the table)